gis/libosmium/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        osmcode libosmium 2.17.0 v
+github.tarball_from archive
+revision            0
+
+categories          gis
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+license             Boost-1 public-domain
+
+description         A fast and flexible C++ library for working with OpenStreetMap data
+
+long_description    Low-level: this is designed to be a building block \
+                    for writing a very customized decoder for a stable \
+                    protobuf schema. If your protobuf schema is \
+                    changing frequently or lazy decoding is not \
+                    critical for your application then this approach \
+                    offers no value: just use the C++ API that can be \
+                    generated with the Google Protobufs protoc \
+                    program.
+
+checksums           rmd160  3923cb87ea091aa69a23ca2ee517ea438ac9fe56 \
+                    sha256  4a7672d7caf4da3bc68619912b298462370c423c697871a0be6273c6686e10d6 \
+                    size    513404
+
+compiler.cxx_standard 2011
+
+depends_lib         port:zlib \
+                    port:protozero \
+                    port:expat \
+                    port:bzip2 \
+                    port:proj8
+
+cmake.build_type    Release
```

Comment:
ideally, this hould not be set and the default value by the `cmake` PG should be used, which is `MacPorts`. This will make sure that all the typical compiler flags and such that are set by MacPorts will be honored. In other words, please try and remove this line and see if it still builds.

Reply:
It builds without specifying that, so I'll remove it.  Thank you.